vizro-core/docs/pages/user_guides/layouts.md

```diff
@@ -398,3 +398,11 @@ Below is a table of examples you can take as a reference to create some selected
 | `layout=Layout(grid=[[0,3],[1,3],[2,3]])`                  | three_left_one_right   | <img src="../../../assets/user_guides/layout/three_left_one_right.png" width="400"/> |
 | `layout=Layout(grid=[[0,0,0],[1,2,3]])`                    | one_top_three_bottom   | <img src="../../../assets/user_guides/layout/one_top_three_bottom.png" width="400"/> |
 | `layout=Layout(grid=[[0,1,2],[3,3,3]])`                    | three_top_one_bottom   | <img src="../../../assets/user_guides/layout/three_top_one_bottom.png" width="400"/> |
+
```

Comment:
Let's also put this section somewhere more prominent towards to the top of the page I think.